### ISSUE TITLE
How to uninstall apps

### ISSUE THREAD

**Issue by JamborJan** [sandstorm-io/sandstorm#829]
How can I remove an app in the new UI? In the past there was an `uninstall` button when the app was chosen. Now I cannot find any way to do that.

**Comment by ocdtrekkie** [comment]
Currently impossible with the UI. It's a known issue.

**Comment by ocdtrekkie** [comment]
A Sandstorm dev can give you some JavaScript to run in console to uninstall an app if you really need it though.

**Comment by JamborJan** [comment]
Ok, the problem is that I have two versions with two different app IDs of the paperwork app (reasons see: https://github.com/JamborJan/paperwork/issues/13) so it is important for me, and also for other users to uninstall an app.

**Comment by ocdtrekkie** [comment]
I experienced that feel before the original uninstall button was added. But yeah, there's a console command you can run, but hopefully uninstall will be added shortly. I think it's one someone's todo list for this week.

**Comment by zarvox** [comment]
I am currently working on UI for this.  Stay tuned. :)

@JamborJan If you know the app ID for the package that you want to remove, then as a workaround in the interim, you can run the following in your JS console while looking at the app list ("New...") page:

```js
UserActions.remove(UserActions.findOne({"appId": "INSERT-YOUR-APPID-HERE"})._id)
Meteor.call("deleteUnusedPackages", "INSERT-YOUR-APPID-HERE")
```

**Comment by JamborJan** [comment]
Thx @zarvox, works of course like a charm.

**Comment by zarvox** [comment]
Fixed by #848.
